src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,22 @@
+import { faGithubAlt } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import("next").NextConfig;
```

Comment:
Should not need to import next for this. Let me know if otherwise

Reply:
This was a terrible copy paste mistake 🫠